patch.py

```diff
@@ -8,12 +8,22 @@
 
 parser = argparse.ArgumentParser(description="This software formats a Dota2' changelog text into HTML.")
 parser.add_argument('--file', '-f', action='store', help="changelog to be formated", required = True, dest = 'file')
+parser.add_argument('--template', '-t', action='store', help="base template to generate HTML", default = 'default', dest = 'template')
 parser.add_argument('--version', '-v', action='version', version='%(prog)s: v1.0 (Yasha)')
 args = parser.parse_args()
 
 #CONSTANT
 CHANGELOG = 'changelogs/'
 
+#Checks if the line starts with fixed and removes it
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->